lib/FrameManager.js

```diff
@@ -197,6 +197,21 @@ class Frame {
 
   /**
    * @param {string} selector
+   * @param {function()|string} pageFunction
+   * @param {!Array<*>} args
+   * @return {!Promise<?ElementHandle>}
```

Comment:
This does not return an element handle. It returns !Promise<*>

Reply:
Done.